e2e/scripts/run-e2e.sh

```diff
@@ -5,8 +5,8 @@ set -euo pipefail
 ENTRY_POINT="${1}"
 TEST="${2}"
 
-export CHAIN_A_TAG="${CHAIN_A_TAG:-latest}"
-export CHAIN_IMAGE="${CHAIN_IMAGE:-ibc-go-simd}"
```

Comment:
the idea behind this one was that this would be the tag you use locally, but that hasn't really been how this has been used, so I think these defaults make more sense! 👍